go/sqltypes/type.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Sorry, I meant file not package. A simple "This file provides conversion functions from sql types to vitess types" would make it easier to understand the purpose of this specific file.